spec/rubocop/cop/rails/file_path_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::FilePath do
+  subject(:cop) { described_class.new }
+
+  context 'when Rails.root.join with some path strings' do
+    let(:source) { 'Rails.root.join(\'app\', \'models\', \'user.rb\')' }
```

Comment:
I'd use "" to eliminate the need to escape ' inside the string. We should actually have a cop for this! :-) /cc @Drenmi @pocke